starter-code/index.html

```diff
@@ -1,31 +1,66 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <title>Instagram</title>
+  <link rel="stylesheet" type="text/css" href="css/main.css">
 </head>
+
 <body>
-  Instagram
+  <div class="header">
+  </div>
 
-  Capture and Share the World's Moments
+  <div class="content">
+    <div class="phonesBox">
+      <img class="twoPhones" src="./images/phones.png" alt="two-phones" />
+    </div>
 
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
+    <div class="textBox">
 
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
+      <img class="instagramLogo" src="./images/brand.png" alt="instagram-logo" />
 
-  Oh yeah, did we mention it's free?
+      <button class="logInButton">
+        <img src="./images/home.png" alt="log-in"> Log in
+      </button>
 
-  Download on the App Store
-  Get it on Google Play
+      <div class="innerBox">
+        <h2 class="subHeader">Capture and Share
+          <br>the World's Moments
+        </h2>
+        <div class="text">
+          <p>Instagram is a
+            <b>fast</b>,
+            <b>beautiful</b> and
+            <b>fun</b> way to share your life with friends and family.
+            <br>
+          </p>
+          <p>Take a picture or video, choose a filter to transform its look
+            <br>and feel, then post to Instagram &mdash; it's that easy. You can
+            <br> even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
+            <br>
+            <p>Oh yeah, did we mention it's free?</p>
```

Comment:
it's not a very good habit to have a lot of <br> tags in your HTML. I checked your code and it will look exactly the same without the br tags.